lib/config/index.js

```diff
@@ -212,4 +214,22 @@ function checkClusterOptions (config) {
   assert(typeof cfg.ip === 'string', '[FATAL] kuzzlerc.cluster.ip: value must be a string');
 }
 
+function preprocessHttpOptions (config) {
+  const cfg = config.http;
+
+  if (cfg === undefined) {
+    return;
+  }
+
+  if (typeof cfg.accessControlAllowOrigin === 'string') {
+    cfg.accessControlAllowOrigin =
+      cfg.accessControlAllowOrigin
+        .split(',')
+        .map(value => value.trim());
+  }
+
+  config.internal.allowAllOrigins = cfg.accessControlAllowOrigin.includes('*'); // Stored to avoid doing includes multiple times later
+  config.internal.cookieAuthentication = !cfg.accessControlAllowOrigin.includes('*'); // If there is no wild card, cookieAuthentication is enabled
+}
```

Comment:
```suggestion
function preprocessHttpOptions (config) {
  const httpConfig = config.http;

  if (httpConfig === undefined) {
    return;
  }

  if (typeof httpConfig.accessControlAllowOrigin === 'string') {
    httpConfig.accessControlAllowOrigin =
      httpConfig.accessControlAllowOrigin
        .split(',')
        .map(value => value.trim());
  }

  // Stored to avoid doing includes multiple times later
  config.internal.allowAllOrigins = httpConfig.accessControlAllowOrigin.includes('*');

  // If there is no wild card, cookieAuthentication is enabled
  config.internal.cookieAuthentication = !httpConfig.accessControlAllowOrigin.includes('*');
}
```